shared/constraints.py

```diff
@@ -31,5 +29,28 @@ def is_active(self, x: np.ndarray) -> bool:
 
 
 @dataclass
-class SimplexConstraint(Constraint):
-    c: SimplexCallable
+class LinearCallable:
+    """Specific callable to SimplexProblem. Keeps A and b accessible."""
```

Comment:
(minor) not necessarily SimplexProblem, will also be relevant for QuadraticProblem